### ISSUE TITLE
The processing and Thinking running again and again

### ISSUE THREAD

**Issue by White0000000003** [qunash/chatgpt-advanced#216]
![Screenshot 2023-08-24 113359](https://github.com/interstellard/chatgpt-advanced/assets/143059424/57dcda3c-fde2-4d4c-9990-28f764f8fd9f)
 
 Did anyone have this issue? 
 They was 110 when I post this

**Comment by ZiaChiu** [comment]
Yes, me too. It used up my 600 requests per hour. And its progress bar, in my case only shows 14%.
